spec/validations_spec.cr

```diff
@@ -199,13 +199,25 @@ describe Avram::Validations do
   describe "validate_inclusion_of" do
     it "validates" do
       allowed_name = attribute("Jamie")
-      Avram::Validations.validate_inclusion_of allowed_name, in: ["Jamie"]
+      Avram::Validations.validate_inclusion_of(allowed_name, in: ["Jamie"])
       allowed_name.valid?.should be_true
 
       forbidden_name = attribute("123123123")
-      Avram::Validations.validate_inclusion_of forbidden_name, in: ["Jamie"]
+      Avram::Validations.validate_inclusion_of(forbidden_name, in: ["Jamie"])
       forbidden_name.errors.should eq(["is invalid"])
     end
+
+    it "can allow nil" do
+      nil_name = attribute(nil)
```

Comment:
I think there is a way to keep the old type signature with just one method. The problem before was that setting the value to “nil” meant the type of the attribute was just nil. What you could do instead is create an attribute with a specific type of String? That way it will accept an array of strings for the allowed value. You may have to manually create the attribute or change the “attribute” helper to accept a type signature it can use for creating the attribute

Does that make sense?

Reply:
That does the trick. Thanks!